src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -350,6 +354,363 @@ def output_gds(self, value: str) -> None:
             raise TypeError("output_gds must be a str")
         self.attr_setter("_output_gds", value)
 
+    @property
+    def output_netlist(self) -> str:
+        """
+        Get the path to the output netlist file.
+
+        :return: The path to the output netlist file.
+        """
+        try:
+            return self.attr_getter("_output_netlist", None)
+        except AttributeError:
+            raise ValueError("Nothing set for the path to the output netlist file yet")
+
+    @output_netlist.setter
+    def output_netlist(self, value: str) -> None:
+        """Set the path to the output netlist file."""
+        if not (isinstance(value, str)):
+            raise TypeError("output_netlist must be a str")
+        self.attr_setter("_output_netlist", value)
+
+    @property
+    def power_nets(self) -> List[str]:
+        """
+        Get the list of power.
```

Comment:
Seems like this section needs to be regenerated?

Reply:
Yes, I will update.

Reply:
Done